DNN Platform/Dnn.AuthServices.Jwt/Package/01.00.00.SqlDataProvider

```diff
@@ -0,0 +1,172 @@
+/************************************************************/
```

Comment:
Should `Library.build` clean up the `Package` folder?  Or should it be added to .gitignore`?